server/methods/core/orders.js

```diff
@@ -971,6 +995,73 @@ export const methods = {
     });
 
     Hooks.Events.run("onOrderRefundCreated", orderId);
+  },
+
+  /**
+   * orders/refunds/returnItems
+   *
+   * @summary Apply a refund to an already captured order
+   * @param {String} orderId - order object
+   * @param {Object} paymentMethod - paymentMethod object
+   * @param {Object} returnItemsInfo - info about return items
+   * @return {null} no return value
+   */
+  "orders/refunds/returnItems": function (orderId, paymentMethod, returnItemsInfo) {
+    check(orderId, String);
+    check(paymentMethod, Reaction.Schemas.PaymentMethod);
+    check(returnItemsInfo, Object);
+
+    // REVIEW: For marketplace implementations, who can refund? Just the marketplace?
+    if (!Reaction.hasPermission("orders")) {
+      throw new Meteor.Error("access-denied", "Access Denied");
+    }
+
+    const fut = new Future();
```

Comment:
Why are we using this here?

Reply:
We are calling `orders/refunds/create` method in this method, its asynchronous and we want the results of, `orders/refunds/create` returned in the method.

Reply:
Ok, I don't think we need to do that anymore, pretty sure that's deprecated (but I would need to check) but I don't think it's hurting anything now as long as it works.